src/components/ui/toast-provider.tsx

```
@@ -47,11 +51,35 @@ export function ToastProvider({ children }: { children: React.ReactNode }) {
+                  {t.action.label}
+                </button>
+              )}
+            </div>
           </div>
         ))}
       </div>
     </ToastContext.Provider>
   )
 }
 
+function ToastEventBridge({ onNotify }: { onNotify: (message: string, opts?: any) => void }) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)